src/components/navBar.tsx

```diff
@@ -79,7 +93,14 @@ export default function NavBar() {
           colorScheme="f3"
           aria="share on socials"
           value="share"
-          onClick={handleNavButtonClick}
+          onClick={(e) => {
```

Comment:
Actually, since you use this function structure twice I would declare it above and have the modal be an input